utils/noaslr/libnoaslr/Cargo.toml

```diff
@@ -14,3 +14,4 @@ crate-type = ["dylib"]
 anyhow = { version = "1.0.71", default-features = false }
 ctor = { version = "0.2.2", default-features = false }
 nix = { version = "0.26.2", default-features = false, features = ["process", "personality"] }
+libc = "0.2"
```

Comment:
Should this be in
```toml
[target.'cfg(target_os = "freebsd")'.dependencies]
```

Reply:
sure, only few oses down the road would apply anyway.